livestyled/client.py

```diff
@@ -428,3 +430,72 @@ def create_news(
             payload
         )
         return NewsSchema().load(new_team)
+
+    def get_merchant_account(
```

Comment:
can we put this in a single line please. This is feeling like we are doing PHP braces all over again. 

`get_merchant {

}`. 

If we have a very long list of parameters, then normally you can break the last bit before the max character length.

Reply:
Same with other methods below.